README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
+[Apache Beam](https://beam.apache.org/), [BigQuery](https://cloud.google.com/bigquery)
 
 ## Getting started
-TODO
+
+Start from [setting up a development environment](#installing-bigflow). 
+Next, go through [the BigFlow tutorial](./docs/tutorial.md). 
+
+Finally, you can dig into details about [project setup and build](./docs/project_setup_and_build.md), [deployment](./docs/deployment.md),
```

Comment:
I think this paragraph is redundant. It duplicates `The main features` section.

Reply:
removed